help-me.js

```diff
@@ -32,18 +30,17 @@ function helpMe (opts) {
       args = [opts.help]
     }
 
-    var out = through()
-    var gs = globStream([opts.dir + '/**/*' + opts.ext])
-    var re = new RegExp(args.map(function (arg) {
+    const out = new PassThrough()
+    const re = new RegExp(args.map(function (arg) {
```

Comment:
Just because I'm trying to improve my knowledge.
Would it be different in here to use an arrow function?